test/FunctionalTests/Microsoft.OData.Core.Tests/UriParser/HardCodedTestModel.cs

```diff
@@ -592,6 +592,10 @@ internal static IEdmModel GetEdmModel()
             var FullyQualifiedNamespaceGetSomeAddressAction = new EdmFunction("Fully.Qualified.Namespace", "GetSomeAddress", FullyQualifiedNamespaceAddressTypeReference, false, null, true /*isComposable*/);
             model.AddElement(FullyQualifiedNamespaceGetSomeAddressAction);
 
+            var FullyQualifiedNamespaceGetSomeAddressFromPersoFunction = new EdmFunction("Fully.Qualified.Namespace", "GetSomeAddressFromPerson", FullyQualifiedNamespaceAddressTypeReference, true, null, true /*isComposable*/);

```

Comment:
nit: Should probably be "PersonFunction"? The "n" at the end of "Person" is missing.

Reply:
#Resolved.